docs/Plugin-sap.commerce.build.md

```diff
@@ -31,14 +39,33 @@ hybris {
    
    //Default task dependencies for all tasks generated by the hybris ant rule
    antTaskDependencies = []
+   
+   //Control the sparse platform bootstrap.
+   //  When enabled, the commerce extensions are extracted from the distribution zip on the need basis.
+   //  Only extensions that are actually used in the project (either directly mentioned in the localextensions.xml or used by other
+   //  used extensions) are extracted.
+   //  The platform itself is always extracted.
+   //  When this mode is enabled, the bootstrapInclude configuration property is ignored.
+   sparseBootstrap {
+      // default is disabled
+      enabled = true
+      // list of extensions that are always extracted, default is an empty list
+      alwaysIncluded = [
+         "yempty",
+         "yocc",
+         "ybackoffice"
+      ]
+   }
 }
 
 dependencies {
     //SAP Commerce Platform Dependencies.
     //  Define zip file dependencies that are unpacked into the project root folder.
     //  You can define more than one zip file to unpack.
     //  `bootstrapInclude` and `bootstrapExclude` define what is unpacked.
+    //  "de.hybris.platform:hybris-commerce-suite:${hybris.version}@zip" is added as default dependency when the plugin is not in CCV2 mode.
     hybrisPlatform "de.hybris.platform:hybris-commerce-suite:${hybris.version.get()}@zip"
+    hybrisPlatform "de.hybris.platform:hybris-commerce-integrations:2211.3@zip"
```

Comment:
Should we introduce a property for the integration package as well?

Reply:
Can be done.

Reply:
Done